src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
Or it can be done in the next way:
const result = people.reduce((acc, person) => {
    if (person.sex === 'm' && (!century || Math.ceil(person.died / 100) === century)) {
      acc.count++;
      acc.sum += person.died - person.born;
    }
    return acc;
  }, {count: 0, sum: 0});

  return result.sum / result.count;